include/retdec/fileformat/types/resource_table/resource_icon.h

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file include/retdec/fileformat/types/resource_table/resource_icon.h
+ * @brief Class for one resource icon.
+ * @copyright (c) 2017 Avast Software, licensed under the MIT license
+ */
+
+#ifndef RETDEC_FILEFORMAT_TYPES_RESOURCE_TABLE_RESOURCE_ICON_H
+#define RETDEC_FILEFORMAT_TYPES_RESOURCE_TABLE_RESOURCE_ICON_H
+
+#include "retdec/fileformat/types/resource_table/resource.h"
+namespace retdec {
+namespace fileformat {
+
+/**
+ * One resource icon
+ */
+class ResourceIcon : public Resource
+{
+    private:
+        unsigned short width;             ///< icon width
```

Comment:
Why is `unsigned short` used here (and also line below this one) but others use fixed-width integers?

Reply:
@metthal because the rest of the attributes correspond to the data found in Icon entry. In icon entry the width and height is represented by 8bit unsigned value, however value 0 represents 256px width/height so i needed a data type that could store the value 256. Using `uint16_t` would be possible but i found it misleading since the data is in entry is not of 2B size. please see `ResourceIcon::setWidth`